specification/postgresqlhsc/resource-manager/Microsoft.DBforPostgreSQL/preview/2020-10-05-privatepreview/examples/ServerGroupListByResourceGroup.json

```diff
@@ -110,6 +112,8 @@
               "resourceProviderType": "Meru",
               "availabilityZone": "1",
               "standbyAvailabilityZone": "2",
+              "readReplicas": null,
```

Comment:
'null' is typical value here as example for reference?

Reply:
Yes, Do you think it is better to provode empty array instead of null?